recipes/dgpsi/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "dgpsi" %}
+{% set version = "2.1b0" %}
```

Comment:
This is a beta release and falls under cfep-05: https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Have updated the package on PyPI to v2.1.0 and the version info in meta.yaml file has also been updated. Please review.